page-objects/administration.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 const firstLogin = require('../shared_steps/firstLogin.js');
-const { expect } = require('chai');
+const { expect, assert } = require('chai');
```

Comment:
do not mix expext and assert, what is the default in this project?

Reply:
i've removed assert and replace with expect.